frontend/src/index.css

```diff
@@ -16,3 +16,20 @@ code {
   --dark-blue: #1D6CA5;
   --interactive-grey: #606062;
 }
+
```

Comment:
solutions for 'chakraui integrated' scrollbar seemed less than ideal and equivalent to using css files https://stackoverflow.com/questions/65042380/how-to-add-webkit-scrollbar-pseudo-element-in-chakra-ui-element-react

open to opinions